test.sh

```diff
@@ -20,46 +20,46 @@ USER_1="2b9046ac-6112-11eb-ae07-3e22fb0d93ba"
 USER_2="38c15834-6112-11eb-86fb-3e22fb0d93ba"
 
```

Comment:
Since you're changing this script anyway, can you make it so that if HOST and PORT are already set as environment variables, they are used as is?

I've been doing this locally:
```
PORT=${PORT:-8000}
```
I think 
```
HOST=${HOST:-"localhost:${PORT}"}
```
would work but I haven't been using it.  These make it easier to use the test script in more than one way, such as running against a PaaS deployment.

Reply:
Good idea.  Added in b508288 and tested locally.